include/json/value.h

```diff
@@ -630,7 +630,7 @@ Json::Value obj_value(Json::objectValue); // {}
 
     void setComment(const char* text, size_t len);
 
-    char* comment_{nullptr};
+    char* comment_{ nullptr };
```

Comment:
I don't really like this behavior, but it's what our .clang-format specifies so we'd better either accept it or change our .clang-format file.

Reply:
I have no preference, except for consistency.  The status-quo is fine.

I'll also use clang-format in the future to maintain style consistency.